src/main/java/baritone/process/MineProcess.java

```diff
@@ -420,6 +421,15 @@ private static List<BlockPos> prune(CalculationContext ctx, List<BlockPos> locs2
                     }
                 })
 
+                .filter(pos -> {
+                    if (Baritone.settings().mineOnlyInsideSelection.value) {
+                        boolean isInSelection = Arrays.stream(ctx.baritone.getSelectionManager().getSelections()).anyMatch(selection -> selection.aabb().contains(new Vec3d(pos.getX(), pos.getY(), pos.getZ())));
+
+                        return !Baritone.settings().mineOnlyInsideSelection.value || isInSelection;
```

Comment:
```suggestion
                        return isInSelection;
```

Reply:
It won't ever be false inside this if unless someone changed it on another thread.

Reply:
Thanks for pointing that out, that's just a remnant from how I was doing it